src/main/kotlin/ecommerce/repository/JdbcCartStatRepository.kt

```diff
@@ -0,0 +1,59 @@
+package ecommerce.repository
+
+import ecommerce.dto.ActiveMemberStat
+import ecommerce.dto.TopProductStat
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.stereotype.Repository
+
+@Repository
+class JdbcCartStatRepository(
+    private val jdbcTemplate: JdbcTemplate,
+) : CartStatRepository {
+    override fun findTop5MostAddedProductsInLast30Days(): List<TopProductStat> {
+        val sql =
+            """
+                    SELECT 
+                ci.product_id,
+                p.name AS product_name,
+                COUNT(*) AS added_count,
+                MAX(ci.added_at) AS last_added_at
+            FROM cart_items ci
+            JOIN products p ON ci.product_id = p.id
+            WHERE ci.added_at >= DATEADD('DAY', -30, CURRENT_TIMESTAMP)
+            GROUP BY ci.product_id, p.name
+            ORDER BY added_count DESC, last_added_at DESC
+            LIMIT 5
+
```

Comment:
```suggestion
            SELECT 
                ci.product_id,
                p.name AS product_name,
                COUNT(*) AS added_count,
                MAX(ci.added_at) AS last_added_at
            FROM cart_items ci
            JOIN products p ON ci.product_id = p.id
            WHERE ci.added_at >= DATEADD('DAY', -30, CURRENT_TIMESTAMP)
            GROUP BY ci.product_id, p.name
            ORDER BY added_count DESC, last_added_at DESC
            LIMIT 5
```

Let's keep the indentation consistent and remove unnecessary line.
Please check other queries in the project and apply this where it's needed 👍